build.gradle

```diff
@@ -16,6 +17,9 @@ allprojects {
     repositories {
         maven { url "https://repository.exoplatform.org/public/" }
         mavenLocal()
+        maven {
+            url "https://maven.google.com"
```

Comment:
We should only use our repository.
@thomasdelhomenie, could you open the ticket to add this repo on it if meeded ?
thanks

Reply:
Good catch.
@fighterpoul is this repository really mandatory ?

Reply:
Yes, it's needed according to official doc: https://firebase.google.com/docs/android/setup

Reply:
ok, thanks.

Reply:
This should not be the case anymore because I added Google Maven Repo in our maven proxy https://repository.exoplatform.org/public/ which is already declared. Please remove the direct reference to Google repo.